src/EnergyPlus/SetPointManager.cc

```diff
@@ -3919,6 +4256,11 @@ void InitSetPointManagers(EnergyPlusData &state)
     //                          SetpointManager:MultiZone:Humidity:Maximum
     //                      Sep 2010 B.A. Nigusse, FSEC/UCF
     //                         Added control varibles for SetpointManage:Scheduled
+    //                      Jan 2022 Wooyoung Jung, Jeremy Lerond and Jian Zhang, PNNL
+    //                         Added new setpoint managers:
+    //                          SetpointManager:SystemNodeReset:Temperature
+    //                          SetpointManager:SystemNodeReset:Humidity
```

Comment:
This is fine, but for reference, we rely more on the Git history to better understand the changes to files in the context of Github PRs and issues.